pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -84,6 +84,10 @@ type NodeResourcesFitArgs struct {
 	// with "example.com", such as "example.com/aaa" and "example.com/bbb".
 	// A resource group name can't contain '/'.
 	IgnoredResourceGroups []string
+
+	// EnablePodOverhead is set during conversion to the internal arg type, based on whether the
+	// PodOverhead feature gate is enabled
+	EnablePodOverhead bool
```

Comment:
Hmm... I'm not sure I like this. If we do this, we have to remove this in a future version when the featuregate reaches GA.

Is moving `features.Constants` to a k8s.io/xyz repo an alternative?

Reply:
> BTW: Is moving features.Constants to a k8s.io/xyz repo an alternative?

That's a good question, and one we started working on (and I think actually got some agreement to move forward) in https://github.com/kubernetes/kubernetes/issues/90136. I reopened that issue to get it moving again because it stalled out